etabotsite/etabotsite/settings.py

```diff
@@ -74,7 +73,7 @@
     
 ]
 
-CORS_ORIGIN_ALLOW_ALL = True   
+# CORS_ORIGIN_ALLOW_ALL = True   
```

Comment:
Why is it commented away?

Reply:
this is for security to forbid cross-origin requests